Dockerfile

```diff
@@ -1,22 +1,22 @@
 #
 # Dockerfile for Storm Mesos framework
 #
-FROM mesosphere/mesos:0.22.0-1.0.ubuntu1404
+FROM mesosphere/mesos:0.25.0-0.2.70.ubuntu1404
 MAINTAINER Timothy Chen <tnachen@gmail.com>
 
 # build packages
 ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update
-RUN apt-get install -yq openjdk-7-jdk
+RUN apt-get install -yq openjdk-7-jdk maven wget
 
 # export environment
 ENV JAVA_HOME /usr/lib/jvm/java-7-openjdk-amd64
 
 ENV MESOS_NATIVE_JAVA_LIBRARY /usr/lib/libmesos.so
 
-# copy local checkout into /opt/storm
-ADD . /opt/storm
+ADD . /work
 
+WORKDIR /work
+RUN ./bin/build-release.sh main
```

Comment:
`ENTRYPOINT` was removed because it makes it possible to pass environment variables via shell commands. With `ENTRYPOINT`, you can pass additional arguments, but they're passed as literal strings (so passing an env var doesn't work).